src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java

```diff
@@ -1561,7 +1561,8 @@ private static String profileArgsDesc(String method, List<String> args) {
             named = true,
             doc =
                 "Working directory for command execution.\n"
-                    + "Can be relative to the repository root or absolute."),
+                    + "Can be relative to the repository root or absolute.\n"
+                    + "The default is relative to the repository root."),
```

Comment:
Maybe just "The default is the repository root."?

Reply:
Done!